### PR TITLE
README formatting and links correction

### PR DESCRIPTION
**Pull request by Inoi-K**
- formatted colons and dots
- udp corrected
- fixed broken links in the "Implementation Details" section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,11 +74,11 @@ fortio.exe server
 ```
 (at the prompt, allow the windows firewall to let connections in)
 
-Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
+Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio](http://localhost:8080/fortio).
```
Comment (ldemailly) [inline README.md]: there is a reason for the trailing / - if you try you'll find out that without the trailing / you get one extra unnecessary redirect

File: README.md [hunk 2]
```diff
@@ -74,11 +74,11 @@ fortio.exe server
...
 
-You can get a preview of the reporting/graphing UI at [https://demo.fortio.org/](https://demo.fortio.org/).
+You can get a preview of the reporting/graphing UI at [https://demo.fortio.org](https://demo.fortio.org).
 <!--
-and on [istio.io/docs/performance-and-scalability/synthetic-benchmarks/](https://istio.io/docs/performance-and-scalability/synthetic-benchmarks/)
+and on [istio.io/docs/performance-and-scalability/synthetic-benchmarks](https://istio.io/docs/performance-and-scalability/synthetic-benchmarks)
```
Comment (ldemailly) [inline README.md]: same

File: README.md [hunk 3]
```diff
@@ -486,7 +486,7 @@ All done 100000 calls (plus 0 warmup) 0.049 ms avg, 80495.0 qps
 ```
 
 ### UDP
-Start the udp-echo server alone and run a load (use `tcp://` prefix for the load test to be for tcp echo server)
+Start the udp-echo server alone and run a load (use `udp://` prefix for the load test to be for udp echo server):
```
Comment (ldemailly) [inline README.md:489]: thx!

File: README.md [hunk 4]
```diff
@@ -1045,14 +1045,14 @@ Fortio X.Y.Z proxy for [::1]:8080 server listening on [::1]:8889
 
 ## Implementation details
 
-Fortio is written in the [Go](https://golang.org) language and includes a scalable semi log histogram in [stats.go](stats/stats.go) and a periodic runner engine in [periodic.go](periodic/periodic.go) with specializations for [http](http/httprunner.go) and [grpc](fortiogrpc/grpcrunner.go).
-The [http/](http/) package includes a very high performance specialized http 1.1 client.
+Fortio is written in the [Go](https://golang.org) language and includes a scalable semi log histogram in [stats.go](stats/stats.go) and a periodic runner engine in [periodic.go](periodic/periodic.go) with specializations for [http](fhttp/httprunner.go) and [grpc](fgrpc/grpcrunner.go).
+The [fhttp/](fhttp/) package includes a very high performance specialized http 1.1 client.
```
Comment (ldemailly) [inline README.md:1049]: thx!

File: README.md [hunk 5]
```diff
@@ -1117,15 +1117,15 @@ Code 200 : 300000
 Response Body Sizes : count 300000 avg 0 +/- 0 min 0 max 0 sum 0
 </pre></details>
 
-Or you can get the data in [JSON format](https://github.com/fortio/fortio/wiki/Sample-JSON-output) (using `-json result.json`)
+Or you can get the data in [JSON format](https://github.com/fortio/fortio/wiki/Sample-JSON-output) (using `-json result.json`).
 
 ### Web/Graphical UI
 
-Or graphically (through the [http://localhost:8080/fortio/](http://localhost:8080/fortio/) web UI):
+Or graphically (through the [http://localhost:8080/fortio](http://localhost:8080/fortio/) web UI):
```
Comment (ldemailly) [inline README.md]: same (trailing / is useful/needed)

**Review by ldemailly — CHANGES_REQUESTED** [review]
Thanks for the PR - just a few issues:

File: README.md (hunk 1 above)
Comment (ldemailly) [inline README.md]:
```suggestion
Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/).
```

File: README.md (hunk 2 above)
Comment (ldemailly) [inline README.md]:
```suggestion
and on [istio.io/docs/performance-and-scalability/synthetic-benchmarks/](https://istio.io/docs/performance-and-scalability/synthetic-benchmarks/)
```

File: README.md (hunk 5 above)
Comment (ldemailly) [inline README.md]:
```suggestion
Or graphically (through the [http://localhost:8080/fortio/](http://localhost:8080/fortio/) web UI):
```

File: README.md [hunk 6]
```diff
@@ -74,9 +74,9 @@ fortio.exe server
 ```
 (at the prompt, allow the windows firewall to let connections in)
 
-Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
+Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/).
```
Comment (ldemailly) [inline README.md]: These `.`s at the end of a sentence while technically correct, they prevent easy copy paste of the correct url, so undoing some.

```suggestion
Once `fortio server` is running, you can visit its web UI at [http://localhost:8080/fortio/](http://localhost:8080/fortio/)
```

File: README.md [hunk 7]
```diff
@@ -74,9 +74,9 @@ fortio.exe server
...
 
-You can get a preview of the reporting/graphing UI at [https://demo.fortio.org/](https://demo.fortio.org/).
+You can get a preview of the reporting/graphing UI at [https://demo.fortio.org](https://demo.fortio.org).
```
Comment (ldemailly) [inline README.md]:
```suggestion
You can get a preview of the reporting/graphing UI at [https://demo.fortio.org](https://demo.fortio.org)
```

File: README.md [hunk 8]
```diff
@@ -1139,7 +1139,7 @@ Code 429 : 56 (1.9 %)
 Code 503 : 15 (0.5 %)
 ```
 
-There are live examples on [demo.fortio.org](https://demo.fortio.org/)
+There are live examples on [demo.fortio.org](https://demo.fortio.org/).
```
Comment (ldemailly) [inline README.md]:
```suggestion
There are live examples on [https://demo.fortio.org](https://demo.fortio.org/)
```

**Review by ldemailly — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Inoi-K) [reply README.md]: oh I see, sorry. Thanks for the info!

File: README.md (hunk 6 above)
Comment (Inoi-K) [reply README.md]: Makes sense 👍🏻
My bad

**Pull request merged**